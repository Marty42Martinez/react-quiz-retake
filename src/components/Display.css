.Display {
  display: flex;
  flex-direction: column;
  align-items: center;

  img {
    height: 350px;
  }

  p {
    font-size: 3.5em;
    font-weight: bold;
  }
}