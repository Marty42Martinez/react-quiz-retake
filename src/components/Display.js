import React from 'react';
import PropTypes from 'prop-types';

function Display({ headerText, footerText, image }) {
  const style = { };
  return (
    <section>
      <p>{headerText}</p>
      <img src={image} />
      <p>{footerText}</p>
    </section>
  );
}

Display.propTypes = {
  headerText: PropTypes.string.isRequired,
  footerText: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};

export default Display;
