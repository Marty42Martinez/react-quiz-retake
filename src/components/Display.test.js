import React from 'react';
import { shallow } from 'enzyme';
import Display from './Display';


describe('Display component', () => {
  it('renders a Display', () => {
    const wrapper = shallow(<Display headerText='hello' footerText='bye' image='img.jpg' />);
    expect(wrapper).toMatchSnapshot();
  });
});
