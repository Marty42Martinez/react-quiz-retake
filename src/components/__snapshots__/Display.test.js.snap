// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component renders a Display 1`] = `
<section
  className="Display"
>
  <p>
    hello
  </p>
  <img
    src="img.jpg"
  />
  <p>
    bye
  </p>
</section>
`;
